opendj-doc-generated-ref/pom.xml

```diff
@@ -185,21 +185,11 @@
                             </extendedClasspath>
                             <outputDir>${project.build.directory}/docbkx-sources/man-pages</outputDir>
                             <tools>
-                                <tool>
-                                    <name>addrate</name>
-                                    <application>com.forgerock.opendj.ldap.tools.AddRate</application>
-                                    <trailingSectionPaths>
-                                        <trailingSectionPath>exit-codes-0-80-89.xml</trailingSectionPath>
-                                        <trailingSectionPath>addrate-examples.xml</trailingSectionPath>
-                                    </trailingSectionPaths>
-                                </tool>
-
```

Comment:
I would like to have some sort of comment in place of deleted CPPL content. E.g.

```xml
<!-- TODO add addrate documentation -->
```

And if it is not necessary, then do not remove the whole section... maybe just the CPPL content?

Reply:
I created empty man page files with CDDL license. So this change has been reverted.